### DISCUSSION TITLE
Hospitalizations/ICU Admissions/Deaths among vaccinated

### DISCUSSION THREAD

**Discussion by jesskrich** [owid/covid-19-data#2200] [General]
Is there any good reason why this data is missing from the dataset? It's probably one of the most important things to know, especially when making the case for the vaccine.

**Comment by lucasrodes** [comment]
Hi @jesskrich,
Thanks for your message.

At this stage, we do not have an automated process for generating this data. This may change, in the future.


For now, I can point you to these files: https://github.com/owid/notebooks/tree/main/EdouardMathieu/2021_11_deaths_by_vax_status/output/.  We recently started collecting this data for a subset of countries (Chile, England, Switzerland, US), as it is not always easily accessible. Note that the file format and/or file URLs might change. The column `Year` refers to the number of days that have passed since 2021-01-01.

I hope this helps.
